specs/validate_drp/cfht_ugrizy/base.yaml

```diff
@@ -1,197 +1,141 @@
 ---
-name: 'cfht'
-base: 'AD1_design.srd'
+# Select the CFHT test dataset
+id: 'cfht'
 metadata_query:
   instrument: 'CFHT'
+  dataset_repo_url: 'https://github.com/lsst/validation_data_cfht.git'
+  dataset_name: 'validation_data_cfht'
+  visits: [849375, 850587]
+  ccds: [12, 13, 14, 21, 22, 23]
```

Comment:
I am really concerned about putting actual visits and ccds in the spec.  I'd much rather specify a tag on the dataset that specifies the subset to be used.  That tag can have a central definition.  Since we don't actually use this info (as far as I know), I'd prefer to leave it out rather than having it in two places that can get out of sync.

I know tags for subsets isn't currently possible, but it will be available in the GEN3 system.

Reply:
Right, I'll open a ticket to revisit this when time comes. At least adding `visits` and `ccds` to query metadata in specs is consistent with adding `filter_name` for now. This is also related with the definition of datasets we need in SQuaSH.